src/molecule/click_cfg.py

```diff
@@ -171,6 +171,16 @@ def all_scenarios(self) -> CliOption:
             default=False,
         )
 
+    @property
+    def all_scenarios_reset(self) -> CliOption:
```

Comment:
Why does this need a different option? Is it just the help text?

Reply:
yeah, i wanted to bring attention to the fact the shared files were being deleted, it's an unfortuante this about click, no way to just revie the help in the decorator b/c it complains about duplicates